include/net/wifi_mgmt.h

```diff
@@ -109,18 +109,22 @@ struct wifi_status {
 };
 
 #include <net/net_if.h>
+#include <net/ethernet.h>
 
 typedef void (*scan_result_cb_t)(struct net_if *iface, int status,
 				 struct wifi_scan_result *entry);
 
-struct net_wifi_mgmt_offload {
+struct net_wifi_mgmt {
```

Comment:
Why the name change? The WiFi configuration API is still offloaded in this solution (we don't have a native supplicant yet).